README.md

```diff
@@ -2516,6 +2517,71 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
 
    
 
+## Backup and restore of Resource Aggregator for ODIM and plugin configurations
```

Comment:
can this be changed to "Backup and restore of  ODIM and Plugin configurations files"

any specific reason to add Resource Aggregator